vindi-woocommerce-subscriptions.php

```diff
@@ -145,6 +145,8 @@ public function __construct()
                 add_action('woocommerce_save_product_variation',
                     array(&$this, 'save_subscription_variation_meta')
                 , 20, 2);
+            } else {
+                add_filter( 'woocommerce_order_item_get_formatted_meta_data', array( &$this, 'hide_item_meta_data'), 10, 1 );
```

Comment:
Só para ajustar o tamanho da linha :smile: 
```suggestion
                add_filter( 'woocommerce_order_item_get_formatted_meta_data',
                    array( &$this, 'hide_item_meta_data'), 10, 1 );
```